### PR TITLE
Added section about tree shaking to TeamsJS topic

### PR DESCRIPTION
**Pull request by JoshuaPartlow**
Section describes support for tree shaking that became available with TeamsJS version 2.31.0

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5242879fbf403de67b835104a047fb08c4632190/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 91 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bbc018d7-77bc-4b29-9570-a333d25ffb6e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by JoshuaPartlow** [comment]
Topic is staged for review and technical input. 

@erikadoyle

File: msteams-platform/tabs/how-to/using-teams-client-library.md [hunk 1]
```diff
@@ -29,6 +29,40 @@ The remainder of this article walks you through the structure and latest updates
 
 TeamsJS v.2.0 introduces the ability for certain types of Teams apps to run across the Microsoft 365 ecosystem. The other Microsoft 365 application hosts (including Microsoft 365 app and Outlook) for Teams apps support a subset of the application types and capabilities you can build for the Teams platform. This support expands over time. For a summary of host support for Teams apps, see [TeamsJS capability support across Microsoft 365](../../m365-apps/teamsjs-support-m365.md).
 
+### Improve app performance with tree shaking
```
Comment (erikadoyle) [inline msteams-platform/tabs/how-to/using-teams-client-library.md:32]: "app performance" kind of sounds like runtime perf. How about:

Improve load time performance with JavaScript tree shaking

File: msteams-platform/tabs/how-to/using-teams-client-library.md [hunk 2]
```diff
@@ -29,6 +29,40 @@ The remainder of this article walks you through the structure and latest updates
...
+
+Starting with TeamsJS version 2.31.0, the library is fully tree-shakeable. Tree shaking is a JavaScript optimization that eliminates unused code. By using tree shaking when an app is bundled for deployment you can reduce package size, which results in faster download and improved load time.
```
Comment (erikadoyle) [inline msteams-platform/tabs/how-to/using-teams-client-library.md:34]: How about (for better flow):

Starting with version 2.31.0, the TeamsJS library is fully tree-shakeable.

And also, always feel free to reference MDN webdocs from MS Learn docs to explain general concepts. (Microsoft has a formal content partnership w/MDN and has a seat on the MDN web docs product advisory board.)

 `[Tree shaking](https://developer.mozilla.org/docs/Glossary/Tree_shaking) is a JavaScript optimization that eliminates unused code.`

**Review by erikadoyle — COMMENTED** [review]
Left a couple minor suggestions, but overall LGTM

File: msteams-platform/tabs/how-to/using-teams-client-library.md (hunk 1 above)
Comment (JoshuaPartlow) [reply msteams-platform/tabs/how-to/using-teams-client-library.md:32]: Updated in next commit.

File: msteams-platform/tabs/how-to/using-teams-client-library.md (hunk 2 above)
Comment (JoshuaPartlow) [reply msteams-platform/tabs/how-to/using-teams-client-library.md:34]: Sounds good, updated in next commit.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2509d67c568187ccc08b0ba98eac277b41176b4d/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 92 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/32ce3fdc-e044-46b3-bc0d-9595e9efb7a1) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by erikadoyle — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-package.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c9f281b73d243a3a63e6cb8a7a14287817164756/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fapps-package.md) | 90 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8eb6685b-9a1e-4baf-a74d-d712526509d5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c9f281b73d243a3a63e6cb8a7a14287817164756/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 92 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/01a27d28-aac1-4b03-a3c4-c2d4c99f0f19) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by ali-ms — APPROVED** [review]
Signing off this from PM side.

**Review by Wajeed-msft — DISMISSED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-package.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bb670b32de6693069ed47b6af230f12f172bdf8b/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fapps-package.md) | 90 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c9ca91b3-e8ea-40f2-a279-d3d47d714e2d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/bb670b32de6693069ed47b6af230f12f172bdf8b/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 92 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dad7a3ff-bbcf-4ec8-852a-377235f42f77) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-package.md](https://github.com/MicrosoftDocs/msteams-docs/blob/161564dbaf6ee568f157a58d91f23a13f00373ed/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fapps-package.md) | 90 | 9 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5635197d-7985-4b0f-93f9-b6ea3e99ceb5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/161564dbaf6ee568f157a58d91f23a13f00373ed/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 92 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d84b445b-69da-473b-8b6f-a202188d8d6c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-package.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e955b1e3c26a07f336fa3f4bd7e5e2e915d837f4/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fapps-package.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/27bab114-792c-43fe-b2df-b1145784f80f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e955b1e3c26a07f336fa3f4bd7e5e2e915d837f4/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 92 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0a273fa0-e75b-434f-b3ad-acb1763fc5d4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-package.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7cd6a1df5fbbbf1b71eddea7a3dd1d5c36baf165/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fapps-package.md) | 92 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/21466239-6c69-4275-b81b-88f4ab351cf2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7cd6a1df5fbbbf1b71eddea7a3dd1d5c36baf165/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-library.md) | 92 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f9d0c0b8-f69a-4c29-b5b4-1aa74a82e19b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-kushals — APPROVED** [review]

**Pull request merged**